### DISCUSSION TITLE
why I can't use nuxt instance isn't a nested/dynamic route ?

### DISCUSSION THREAD

**Discussion by IOU4** [nuxt/nuxt#17488] [Questions]
I've got this structure :
pages
├── index
│   ├── index.vue
│   ├── items
│   │   └── [id].vue
│   └── items.vue
└── index.vue

now when I try to read the route params at `[id].vue` with `useRoute()`, I get an error saying `nuxt instance unavailable`,
however I'm able to use that in `items.vue`
that occurs when I try to use any of my composables too.

nuxt@3.0.0-rc.3
node v18.3.0

**Comment by danya-rnd** [comment]
Where do u use it? Provide the example of code block inside .vue file.

**Reply by IOU4** [reply]
It was just silly me miss typing `setup` I have just realized that when I tried to copy the code inside [id].vue, thanks for replying
